update.sh

```diff
@@ -13,6 +13,34 @@ END
 sysctl -p /etc/sysctl.d/10-ambrosus.conf
 
 git pull origin master
+
+###
+
+ # install or update nvm
```

Comment:
Why this is here "###" and a space before "# install"?

Reply:
It's easier to read. Should be removed after the merge.